website/content/docs/builders/amazon/ebs.mdx

```diff
@@ -91,6 +91,64 @@ necessary for this build to succeed and can be found further down the page.
 
 @include 'builder/amazon/common/RunConfig-not-required.mdx'
 
+#### Metadata Settings
```

Comment:
Nice!! Thanks for making the changes :D 
Don't forget you must add these to the other ebs builder docs as well.